root.C

```diff
@@ -0,0 +1,11 @@
+package main
+import ("fmt")
+
+func main() {
+  var i float64
+
+  fmt.Print("Type a number: ")
+  fmt.Scan(&i)
+  result := math.Sqrt(i)
+  fmt.Println("Your square root is:", i)
```

Comment:
result declared but not used. this program gives an error also please change file name